addons/mail/static/src/components/attachment_link_preview/attachment_link_preview.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="mail.AttachmentLinkPreview" owl="1">
+        <t t-if="attachmentLinkPreviewView">
+            <div class="o_AttachmentLinkPreview rounded px-3 pt-3 pb-2 d-flex" t-attf-class="{{ className }}" t-ref="root">
+                <t t-if="attachmentLinkPreviewView.attachment">
+                    <img t-if="attachmentLinkPreviewView.attachment.mimetype === 'application/o-linkpreview-with-thumbnail'" class="o_AttachmentLinkPreview_linkImage rounded mr-3 mb-2" t-att-src="attachmentLinkPreviewView.imageUrl" t-att-alt="attachmentLinkPreviewView.attachment.name" />
+                    <div t-if="attachmentLinkPreviewView.attachment.name" class="o_AttachmentLinkPreview_content">
+                        <a t-att-href="attachmentLinkPreviewView.attachment.url" class="o_AttachmentLinkPreview_title" target="_blank">
+                            <h3 class="o_AttachmentLinkPreview_linktitle">
+                                <t t-esc="attachmentLinkPreviewView.attachment.name"/>
+                            </h3>
```

Comment:
can do everything inline with `<h3 t-esc=""/>`

Reply:
I m not really a fan, I think it make it harder to find what's inside this `h3`, but ok.

Reply:
This makes it easier to see the whole shape of the template.
XML sucks, but having `t-esc` inline is IMO less worst.